examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -42,15 +42,32 @@ model:
     drop_last: false
     pin_memory: false
 
+
+  tokenizer:
+    tokenizer_name: bert # or sentencepiece
+    pretrained_model_name: ${model.language_model.pretrained_model_name}
+    vocab_file: null # path to vocab file
+    data_file: null # used to build sentencepiece
+    sample_size: null # for sentencepiece
+    vocab_size: null # vocab size for building sentence piece
+    do_lower_case: null # only used when tokenizer is build with vocab file or with sentencepiece  
+    tokenizer_model: null # only used if tokenizer is sentencepiece
+    special_tokens: 
```

Comment:
this is contradictory to your feedback not to use **cfg.tokenizer
If i comment it out, i will get an error calling get_tokenizer(), e.g. if data_file is gone, get_tokenizer(data_file=data_file) will error out. It could have been solved by passing  **cfg.tokenizer, but @VahidooX @ekmb @titu1994  are all against it